libs/ledgerjs/packages/cryptoassets/src/data/exchange/coins.json

```diff
@@ -1 +1 @@
353474212","304402203c5ee34fe17bd801ed6d0da330927eb2a796a786aaf124134cb915c93c108a4802203372a9a47d52ac2c06afab3e618eca44eb6e7fce5ad860825cf41405c13bd1a4"],["stellar","03584c4d075374656c6c617200","30440220601a3941b0aece3fb970566252784343c4f3497f53dba3db8f74b5a66a43ea6f02201be00ca7def64829892e6a9c0affb361d58df2899109d70c039ce527b9ed9e33"],["stellar_testnet","0474584c4d0f5374656c6c617220546573746e657400","30440220213fc64616838003e4a2536bd991a8d4cca06ef46c8ff8c18e2ed1baf836bfab022044d68c200f40d761211c35bbe5265b33f7ca0ee5ddcde6b634e40c2ab325fc97"],["stratis","055354524154075374726174697300","304402203d571457df62e869038b5794024720fddca5f8ebd427958a79896554df833c7e02204914f131a398098146882655a3381fa84a6ebc7cc48744d58512a4d1a512eb6e"],["stride","04535452440653747269646500","3045022100cb0970b79775bb35b19cb1278a760ed2b2e0a880e563da36aa4c511237050ae3022005e68410e0729256d114ecdc105b66287b75ed2b7a49ed85247c942d06451c96"],["syscoin","0353595307537973636f696e050353595312","3044022060f554250c07d952773c07bf63190aaa56034a6cd1e733ef2b98b6cacb8bfc9102203c193387395ab7ebb3a65ef4c3a6922613cabdf5a8cf545a7f65046a61ed0539"],["tezos","0358545a0554657a6f7300","30440220410f671642478312444d3c20c0a6f76329218bcfc9906f2e58720f87dcafba5802200ffbd2ecb4fa1bbb283e81d36cf57857938fad01f2409c6a74b16cee8d327822"],["tomo","04544f4d4f04546f6d6f0604544f4d4f12","3044022042edfd8501902264d1ff11ac7422c7b6f71c7c3d7610d61ca9be36cc8430af1a0220181c14a7efb6c64ba5811cb82fd44a7aecc99c9232e8ef6075fd1c20c54271c7"],["tomo_testnet","0574544f4d4f0c546f6d6f20546573746e6574070574544f4d4f12","3044022057575b9a6c84525b81f71df56504d916f5b0043b67f0cc1a1d3dd134dab4886002207c21d1f04598f51381f57a076a7251f41745cb198331e5eab72cb4c9f7f313e4"],["tron","035452580454726f6e00","3045022100cc308621933e8e94ef6f0a1c05cd6e766a4c4131e0b320cdd8c0f02d7bdb919002206b18b6e92c0c7772be31c61bfbb6879f5e2f2f4e335294184a71e0e1e319c7cb"],["velas_evm","03564c580956656c61732045766d0503564c5812","3044022062902d424e29d1f206f0f342e22e4b19380866a0c6c248ba4c18d34f155147dc0220695442b7e1e17591d5184a950d418d475d367dd6b409a7b0794f6654548acfe1"],["vertcoin","035654430856657274636f696e00","3045022100bdbc975f4734e72986504684cb82e610bd46d2dbd7e628454c37a5afd7ffe1ae02202fbdd7880867632c068ee64f70e4207e2638ccac2d58357770c9b1c28d28cee6"],["viacoin","0356494107566961636f696e00","3044022013297952a9568e41d1c4195c1eafeb8248dfd45a0c55a9ba96f0209581316f6a0220124661c2026a1141be6e17bf56362f9f2892c784a8fc45a2595b28ec9210dd2b"],["westend","0474574e440757657374656e6400","3044022000ae68b4878f91c2d6df6933e287f845a86c54cdbf597518b593588cbfc530d402205713c4ee9da0552f07ba628a5aa41b311620e6e5d360896796298cdd09fc8f54"],["zcash","035a4543055a6361736800","3045022100d6d49d7b51935899672edf3760a36207dbc3172de2cb635d4af5322fab7c669a02206881ca3ad323e1e2334c4838fa9d17f105ecc574153078f578916eeca908d6fc"],["zencash","035a454e07486f72697a656e00","304402202ca9aab0a04fdb69443519d1d528bb890e3f2bd2a0a0842d62cce543ac00e7ed022001ec56b9710adb7ca7509dda6c095cf741fe00f5fbd17a1bb7201626f6337a42"]]
```

Comment:
So I asked Julien about this file. His reply for posterity:
> This file is generated by https://github.com/LedgerHQ/ledger-live/tree/develop/libs/ledgerjs/packages/cryptoassets/src/crypto-assets-importer
It imports from the CAL here: https://github.com/LedgerHQ/crypto-assets/tree/main/assets/coins/osmo
[2:55](https://ledger.slack.com/archives/D07EHCHSK5Z/p1722261333882039)
Problem: once a currency id has been defined in CAL, it is used by B2B and B2C, so changing it could cause many weird issues/bugs
So usually, it would be on downstream consumers to adapt to what has been defined in the CAL (in this case, use osmo everywhere)